template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/resources/README.md.tmpl

```diff
@@ -143,6 +143,41 @@ Model validation contains three components:
 To set up and enable model validation, update [validation.py](../validation/validation.py) to return desired custom metrics and validation thresholds, then 
 resolve the `TODOs` in the ModelValidation task of [model-workflow-resource.yml](./model-workflow-resource.yml).
 
+
+### Setting up model monitoring
```

Comment:
```suggestion
### Setting up Lakehouse monitoring
```

Reply:
I understand we are still discussing on this internally. I will update this when we finalize the unified naming convention